doc/source/timedeltas.rst

```diff
@@ -250,8 +248,8 @@ Note that division by the NumPy scalar is true division, while astyping is equiv
 
 .. ipython:: python
 
-   td = pd.Series(pd.date_range('20130101', periods=4)) - \
-        pd.Series(pd.date_range('20121201', periods=4))
+   td = pd.Series(pd.date_range('20130101', periods=4)) -\
+       pd.Series(pd.date_range('20121201', periods=4))
```

Comment:
better use brackets than a backslash (and the `-` after the line break)

Reply:
Without \ (within brackets or not) I get either 
* `W504 line break after binary operator`
* `W503 line break before binary operator`

Enclosing it with brackets as is results in `E502 the backslash is redundant between brackets`

Reply:
I think in `setup.cfg` we are ignoring both of them for `flake8`. We should also ignore `W504` for `flake8-rst`. I think the only reason it exists is because `W504` was discouraged in the past, and now `W503` is discouraged.